youtube_dl/extractor/youtube.py

```diff
@@ -3019,93 +3073,35 @@ def _real_extract(self, url):
 
 class YoutubeSearchIE(SearchInfoExtractor, YoutubeBaseInfoExtractor):
     IE_DESC = 'YouTube.com searches'
-    # there doesn't appear to be a real limit, for example if you search for
-    # 'python' you get more than 8.000.000 results
-    _MAX_RESULTS = float('inf')
     IE_NAME = 'youtube:search'
     _SEARCH_KEY = 'ytsearch'
-    _SEARCH_PARAMS = None
+    _SEARCH_PARAMS = 'EgIQAQ%3D%3D'  # Videos only
+    _MAX_RESULTS = float('inf')
     _TESTS = []
```

Comment:
Should there be a test at least for form's sake? I know there wasn't one and yt-dlp doesn't have one either, but maybe just 
```
    _TESTS = [{
        'url': 'ytsearch10:youtube-dl test video',
        'playlist_count': 10,
        'info_dict': {
            'id': 'youtube-dl test video',
            'title': 'youtube-dl test video',
        }
    }]
```
And the same for `YoutubeSearchDateIE` with `ytsearchdate10:...`?

Reply:
Done

FYI: You can directly commit to PRs

Reply:
As in, me personally? I think I knew that, but protocol.

Reply:
Sure, I don't mind either way.

Just letting you know coz (for me at least) it's often easier to just directly make changes and merge PR rather than review and wait for the other party to respond. As long as it is not some controversial change, I doubt the contributors would mind.